scripts/globals/trust.lua

```diff
@@ -82,3 +82,88 @@ tpz.trust.spawn = function(caster, spell)
 
     return 0
 end
+
+tpz.trust.cipherToSpellID = {
+    [10112] = 906,  -- Zeid
```

Comment:
Yeah, at this point there's going to be 3 places that'll need them, so it may as well happen 🤷‍♂

Reply:
_cries in 'explained this to past authors' again_

subid on the item. then one call to get the subid instead of maintaining big old enums in multiple places.